README.md

```diff
@@ -0,0 +1,35 @@
+## Hamiti - Neue Fische Capstone Project
+
+Hamiti is an app that helps users keep track of their food inventory and recipes.
+I am developing it as a capstone project for Neue Fische Bootcamp.
+
+## Installation
+
+To install use:
+
+```
+yarn install
```

Comment:
We've used `npm` so far so I presume this should look like the following
```suggestion
npm install
```

Reply:
Yarn install is right 😬
I had peer dependency issues between emotion and material UI, so I replaced package-lock.json with yarn.lock.

Reply:
Alright cool, then you need to change the other line to make yarn run the app not npm